src/components/datetime/datetime.ts

```diff
@@ -135,6 +135,9 @@ import {
  * to serialize and pass within JSON objects, and sending databases a standardized
  * format which it can be easily parsed if need be.
  *
+ * To create an ISO datetime string for the current date and time, e.g. use `currentDate: 
+ * String = new Date().toISOString();`.
```

Comment:
Could this be changed to a code block and lowercase string like:

```
 * The following example shows how to create an ISO datetime string for the current date and time.
 *
 * ```ts
 * currentDate: string = new Date().toISOString();
 * ```
```